firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,14 @@
+SCENARIO("Get the byte from expected position", "[bytes]")
+   GIVEN("A Byte '\xff'")
+   WHEN("a byte is fetched ")
+     THEN("value is as expected")
+
+
+SCENARIO("Set byte ")
```

Comment:
This scenario appears to test the `set_byte<byte_index, T>` function; we also need a scenario to test the `set_byte<T>` function (where byte_index is specified as a function parameter rather than a template parameter)